src/style/properties.js

```diff
@@ -356,9 +358,11 @@ const styfn = {};
     { name: 'control-point-weights', type: t.numbers, triggersBounds: diff.any },
     { name: 'segment-distances', type: t.bidirectionalSizes, triggersBounds: diff.any },
     { name: 'segment-weights', type: t.numbers, triggersBounds: diff.any },
+    { name: 'segment-radii', type: t.numbers, triggersBounds: diff.any },
```

Comment:
Let's make this `segment-radius` to be consistent in naming with your other properties.  A lot of the other, existing properties are singular even though they accept multiple values (e.g. `background-image`).

Reply:
I didn't notice you had aliases.  Disregard